components/config-mgmt-service/backend/elastic/nodes.go

```diff
@@ -24,38 +24,25 @@ func init() {
 	}
 }
 
-// GetNode Returns a single Node
-func (es Backend) GetNode(id string) (backend.Node, error) {
-	var node backend.Node
-	filters := map[string][]string{
-		"exists":      []string{"true"},
-		"entity_uuid": []string{id},
-	}
+// NodeExists - Does a node with entity_uuid == 'nodeID' exists
+func (es Backend) NodeExists(nodeID string, filters map[string][]string) (bool, error) {
+	filters["exists"] = []string{"true"}
+	filters["entity_uuid"] = []string{nodeID}
 	filtersQuery := newBoolQueryFromFilters(filters)
 
 	searchResult, err := es.client.Search().
 		Query(filtersQuery).
 		Index(IndexNodeState).
 		Do(context.Background())
 	if err != nil {
-		return node, err
+		return false, err
 	}
 
-	if searchResult.Hits.TotalHits > 0 {
-		// We will expect a single hit to come back from the ES query
-		// and then unmarshal the Source into a backend.Node
-		hit := searchResult.Hits.Hits[0]
-		err = json.Unmarshal(*hit.Source, &node)
-		if err != nil {
-			log.WithFields(log.Fields{
-				"object": hit.Source,
-			}).WithError(err).Debug("Unable to unmarshal the node object")
-		}
-	} else {
-		err = errors.New(errors.NodeNotFound, "Invalid ID")
+	if searchResult.Hits.TotalHits == 0 {
+		return false, nil
 	}
 
-	return node, err
+	return true, err
```

Comment:
`err` is checked a few lines up, so it will always be nil here, right? You could replace this last conditional and returns with one return statement like `return searchResult.Hits.TotalHits == 0, nil`